From 93c6c3bc7541ad169824ac603267e9c02fe4ecd5 Mon Sep 17 00:00:00 2001
From: Christian Benincasa <chrisbenincasa@gmail.com>
Date: Fri, 1 Sep 2023 15:14:02 -0400
Subject: [PATCH 1/5] Set new ORIGINALDATE tag for m4a files in addition to
 existing "ORIGINAL YEAR" tag. The former is used by Plex Media Server to
 discern the original release date. Fixes #70

This behavior also matches other popular tagging libraries, such as https://github.com/Borewit/music-metadata
---
 mediafile.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mediafile.py b/mediafile.py
index 8ff352a..6dbbecb 100644
--- a/mediafile.py
+++ b/mediafile.py
@@ -2087,6 +2087,7 @@ class MediaFile(object):
     original_date = DateField(
         MP3StorageStyle('TDOR'),
         MP4StorageStyle('----:com.apple.iTunes:ORIGINAL YEAR'),
+        MP4StorageStyle('----:com.apple.iTunes:ORIGINALDATE'),
         StorageStyle('ORIGINALDATE'),
         ASFStorageStyle('WM/OriginalReleaseYear'))
 
-- 
2.47.0

